README.md

```diff
@@ -137,18 +124,47 @@ index-servers = pypi, testpypi
 
 ``````
 
-7. Test the package on testpypi with `pip install --index-url https://test.pypi.org/simple/ mypackagename`.
-8. If everything is good, use the Github workflow, `package.yml` to trigger the publishing to PyPI.
+1. Test the package on testpypi with `pip install --index-url https://test.pypi.org/simple/ mypackagename`.
+1. If everything is good, use the Github workflow, `package.yml` to trigger the publishing to PyPI.
 
 ### Instruction for publish to Anaconda
 
 Publishing to Anaconda is handled via workflow, `package.yml`.
 
-Development environment setup
------------------------------
+## Development environment setup
 
 ### Build development environment
 
 1. By default, we recommend providing a single `environment.yml` that covers all necessary packages for development.
 2. The runtime dependency should be in `meta.yaml` for anaconda packaging, and `pyproject.toml` for PyPI publishing.
 3. When performing editable install for your feature branch, make sure to use `pip install --no-deps -e .` to ensure that `pip` does not install additional packages from `pyproject.toml` into development environment by accident.
+
+## Pixi
+
+Pixi is a tool that helps to manage the project's dependencies and environment.
+Currently this template repo have both conventional `conda` based environment (`environment.yml` and `conda.recipe/meta.yaml`) and `pixi` based environment (`pyproject.toml`).
+
+### How to use Pixi
+
+1. Install `pixi` by running `curl -fsSL https://pixi.sh/install.sh | bash`
```

Comment:
Do you want to add a link to their install instructions? Maybe adding a general link to https://pixi.sh/ at the beginning of the section is enough.

Reply:
added the suggested link to readme.